libs/application-generic/src/schemas/control/in-app-control.schema.ts

```diff
@@ -23,7 +23,7 @@ const redirectUrlRegex =
 
 const redirectZodSchema = z.object({
   url: z.string().regex(redirectUrlRegex),
-  target: z.enum(['_self', '_blank', '_parent', '_top', '_unfencedTop']).default('_blank'),
+  target: z.enum(['_self', '_blank', '_parent', '_top', '_unfencedTop']),
```

Comment:
Need to remove this for ajv to work correctly with anyOf. It seemed to be doing nothing anyway, either on creation or update, since placeholder is _self.

Reply:
@LetItRock Btw, do you know why this was there in the first place? It didn't seem to play a role in creation.